public/api-documentation/swagger.json

```diff
@@ -262,6 +262,46 @@
         }
       }
     },
+    "/manifests/{id}/cromerr_submit_transaction": {
```

Comment:
Once again, the awkwardness of the purpose of the service rears its ugly head. "cromerr submit" modifies "transaction" as that's what we're creating in our post. It's a *big*, painful stretch. cromerr_submission_transaction?

Ugh.